# Default configuration for emergency handler
---
/**:
  ros__parameters:
    update_rate: 10
    timeout_hazard_status: 0.5
    use_comfortable_stop: false
